elasticsearch_helper_instant.libraries.yml

```diff
@@ -1,15 +1,3 @@
-#typeahead:
-#  remote: https://typeahead.js.org
-#  version: 1.2.0
-#  license:
-#    name: MIT
-#    url: https://github.com/corejavascript/typeahead.js/blob/master/license
-#    gpl-compatible: true
-#  js:
-#    /libraries/typeahead.js/dist/typeahead.bundle.min.js: { minified: true }
-#  dependencies:
-#    - core/jquery
-
```

Comment:
`instant-search` library below has dependency on `typeahead` library. Is it safe to remove it altogether?

Reply:
Oh, those lines are commented out! Not sure why. Also I wonder if instant search works without the typeahead library.

Reply:
Hmm ... Was wondering the same but then noticed that `typeahead` included in `js/typeahead.bundle.min.js`